DSharpPlus.Commands/Processors/TextCommands/TextCommandConfiguration.cs

```diff
@@ -6,6 +6,10 @@ namespace DSharpPlus.Commands.Processors.TextCommands;
 
 public record TextCommandConfiguration
 {
+    /// <summary>
+    /// The function to use to resolve prefixes for commands.
+    /// </summary>
+    /// <remarks>For dynamic prefix resolving, <see cref="IPrefixResolver"/> registered to the <see cref="DiscordClient"/>'s <see cref="IServiceProvider"/> should be preferred.</remarks>
     public ResolvePrefixDelegateAsync PrefixResolver { get; init; } = new DefaultPrefixResolver(true,"!").ResolvePrefixAsync;

```

Comment:
Perhaps mark this as obsolete?

Reply:
As for obsoletion, both are valid options. w.r.t. Your second comment, see the remark in the docs